### PR TITLE
HTTPS on URL link

### PR DESCRIPTION
**Pull request by x89**

### CODE REVIEW

**Comment by tremby** [comment]
See #4

File: imgur.sh [hunk 1]
```diff
@@ -73,7 +73,7 @@ while [ $# -gt 0 ]; do
 	url="${url%%</link>*}"
 	delete_hash="${response##*<deletehash>}"
 	delete_hash="${delete_hash%%</deletehash>*}"
-	echo $url
+	echo $url | sed 's/^http/https/'
```
Comment (tremby) [inline imgur.sh]: If you just add a colon to the end of this I'll merge it; `s/^http:/https:/`. I know it likely doesn't ever make any practical difference but I like to be sure.

**Comment by tremby** [comment]
Great; squash the commits into one, please.

**Comment by tremby** [comment]
In case you don't know how, `git rebase -i HEAD\~2`, then edit the line with the "Better sed regex" to start with an `f` for fixup (since you just want to keep the other commit message), then save. Then `git push -f`

**Comment by x89** [comment]
That should be it, yeah I'm familiar with commit squashing, just don't tend to take a strong stance on it myself!

Let me know if you'd like anything else, and thanks for the PR, I appreciate it 😘

**Comment by x89** [comment]
Thanks! Hopefully upstream return https as well eventually.

**Comment by tremby** [comment]
The code as you've written it won't mind either way, which is good.

**Pull request merged**